qiskit/qpy/binary_io/schedules.py

```diff
@@ -103,19 +103,25 @@ def _read_discriminator(file_obj, version):
     return Discriminator(name=name, **params)
 
 
-def _loads_symbolic_expr(expr_bytes):
-    from sympy import parse_expr
-
+def _loads_symbolic_expr(expr_bytes, use_symengine=False):
     if expr_bytes == b"":
         return None
+    if use_symengine:
+        _optional.HAS_SYMENGINE.require_now("loads_symbolic_expr")
```

Comment:
I'd probably change this to be:

```suggestion
        _optional.HAS_SYMENGINE.require_now("load a symengine expression")
```

Since the string here gets used in the exception message as the `name` field: https://github.com/ElePT/qiskit-terra/blob/ec0bb10936bfc2bff20444d964f3d42ffec752da/qiskit/exceptions.py#L82